### DISCUSSION TITLE
Some issues and suggestions

### DISCUSSION THREAD

**Discussion by ykla** [helloSystem/hello#331] [Q&A]
For 0.8.0.
Your system is excellent, but there are still some minor problems: the terminal `ctrl c` doesn't act as an interrupt as it should in a normal terminal, his function is actually to copy. For high-resolution screens is missing the function of overall desktop scaling, such as scaling `125%`. In addition, the desktop does not have multi-language support, can you support it, I can help translate. And is it considered to set a separate pkg source or ports for the desktop for maintenance? This will make it easier to upgrade and system installation.

**Comment by probonopd** [comment]
Hello @ykla, thanks for your thoughts. Please do search for existing issues and comment there, or open one new issue per topic in the future.

> the terminal ctrl c doesn't act as an interrupt as it should in a normal terminal, his function is actually to copy. 

As of helloSystem 0.7.0, you need to press SHIFT-Command-C (on PC keyboards, this is SHIFT-Alt-C) because, as you write, we want Command-C to work like it does in all other programs - that is, to copy. Discussion on this topic is in https://github.com/helloSystem/hello/issues/322.

> For high-resolution screens is missing the function of overall desktop scaling, such as scaling 125%.

Currently we can only do 100% for normal screens and 200% for HiDPI screens. While Qt could do scaling at other fractions, I am not sure whether Gtk and other toolkits also could do it.

> In addition, the desktop does not have multi-language support, can you support it, I can help translate.

Actually, helloSystem has multi-language support and I am using it in German, for example. Translation happens here:
https://hosted.weblate.org/projects/hellosystem/

> And is it considered to set a separate pkg source or ports for the desktop for maintenance?

I don't entirely understand the question. But the vast majority of FreeBSD packages comes from the official FreeBSD packages repository. We don't want to build our own packages if it can be avoided.

**Reply by ykla** [reply]
Maybe you should update https://hosted.weblate.org/projects/hellosystem/create-live-media/#alerts 

People cannot continue translate them for this.

**Reply by ykla** [reply]
Not only that, almost all translation components have similar errors.

**Reply by probonopd** [reply]
Thanks for letting me know. Should hopefully be fixed now.
